android/src/main/java/com/sbaiahmed1/reactnativeblur/ReactNativeLiquidGlassView.kt

```
@@ -0,0 +1,487 @@
+   * @param color Hex string (e.g. "#FF0000"), or "clear"/"transparent"/null for no tint
+   */
+  fun setGlassTintColor(color: String?) {
+    glassTintColor =
+      if (color == null || color.equals("clear", ignoreCase = true) ||
+        color.equals("transparent", ignoreCase = true)
+      ) {
+        Color.TRANSPARENT
+      } else {
+        try {
+          color.toColorInt()
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)